core/station/api/src/system.rs

```diff
@@ -59,11 +62,46 @@ pub struct SystemInfoResponse {
 }
 
 #[derive(CandidType, serde::Serialize, Deserialize, Clone, Debug)]
-pub struct AdminInitInput {
+pub struct UserInitInput {
```

Comment:
Maybe InitUserInput to be consistent with InitUserGroupInput?

Reply:
Closing since this was already changed at the time of the comment

Reply:
Ah my bad, I misread your suggestion multiple times.